tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py

```diff
@@ -1,114 +1,280 @@
+                        result_field='OTHER_SANCTION', result_function=validators.oneOf((1, 2)),
+                  ),
+                  validators.if_then_validator(
+                        condition_field='OTHER_TOTAL_REDUCTIONS', condition_function=validators.isLargerThan(0),
+                        result_field='FAMILY_CAP', result_function=validators.oneOf((1, 2)),
+                  ),
+                  validators.if_then_validator(
+                        condition_field='OTHER_TOTAL_REDUCTIONS', condition_function=validators.isLargerThan(0),
+                        result_field='REDUCTIONS_ON_RECEIPTS', result_function=validators.oneOf((1, 2)),
+                  ),
+                  validators.if_then_validator(
+                        condition_field='OTHER_TOTAL_REDUCTIONS', condition_function=validators.isLargerThan(0),
+                        result_field='OTHER_NON_SANCTION', result_function=validators.oneOf((1, 2)),
+                  ),
+                  validators.sumIsLarger(("AMT_FOOD_STAMP_ASSISTANCE", "AMT_SUB_CC", "CC_AMOUNT", "TRANSP_AMOUNT",
+                                          "TRANSITION_SERVICES_AMOUNT", "OTHER_AMOUNT"), 0)
+            ],
+            fields=[
+                  Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
+                        required=True, validators=[]),
+                  Field(item="4", name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8,
+                        required=True, validators=[
+                              validators.month_year_yearIsLargerThan(1998),
+                              validators.month_year_monthIsValid(),
+                        ]),
+                  Field(item="6", name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
+                        required=True, validators=[
+                              validators.isAlphaNumeric(),
+                              validators.notMatches('_'*11),
+                              validators.notEmpty(),
+                        ]),
+                  Field(item="2", name='COUNTY_FIPS_CODE', type='string', startIndex=19, endIndex=22,
+                        required=True, validators=[
+                              validators.isNumber(),
+                        ]),
+                  Field(item="5", name='STRATUM', type='number', startIndex=22, endIndex=24,
+                        required=True, validators=[
+                              validators.isInLimits(0, 99),
+                        ]),
+                  Field(item="7", name='ZIP_CODE', type='string', startIndex=24, endIndex=29,
+                        required=True, validators=[
+                              validators.isNumber(),
+                        ]),
+                  Field(item="8", name='FUNDING_STREAM', type='number', startIndex=29, endIndex=30,
+                        required=True, validators=[
+                              validators.oneOf([1, 2]),
```

Comment:
```suggestion
                              validators.oneOf([1, 2, 3]),
```

Reply:
⬆️ OFA made a change to this data element in DIGIT on 8/10/2023. added another value. also updated in validation workboook.